pkg/kubelet/cm/dra/plugin/plugin.go

```diff
@@ -53,9 +98,11 @@ func (h *RegistrationHandler) RegisterPlugin(pluginName string, endpoint string,
 
 	// Storing endpoint of newly registered DRA Plugin into the map, where plugin name will be the key
 	// all other DRA components will be able to get the actual socket of DRA plugins by its name.
-	draPlugins.Set(pluginName, &Plugin{
+	// By default we assume the supported plugin version is v1alpha3
+	draPlugins.add(pluginName, &plugin{
 		conn:                    nil,
 		endpoint:                endpoint,
+		fallbackVersion:         "v1alpha3",
```

Comment:
Would it make sense to use constant here to avoid changing this every time we bump version?

Reply:
done in done in a57da3914ab